lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -21,16 +21,20 @@
  * included in "SetMessageManagerHandler.h"
  */
 #include "SetMessageManagerHandler.h"
+#include "SessionStateEnforcer.h"
+#include "includes/MagmaService.h"
 #include "ProtobufCreators.h"
 #include "RuleStore.h"
 #include "SessionState.h"
 #include "SessionStore.h"
+#include "includes/ServiceRegistrySingleton.h"
 #include "SessiondMocks.h"
 #include "StoredState.h"
 #include "magma_logging.h"
 #include "PipelinedClient.h"
 #include "AmfServiceClient.h"
 #include "Consts.h"
+#include "EnumToString.cpp"
```

Comment:
is this meant to be EnumToString.h ?

Reply:
Yeah, replaced EnumToString.cpp with EnumToString.h